tests/integration/targets/nxos_bfd_interfaces/tests/common/overridden.yaml

```diff
@@ -13,16 +13,20 @@
   when: platform is not search('N5K|N6K')
 
 - name: setup1
-  ansible.netcommon.cli_config: &id002
+  cisco.nxos.nxos_config: &id002
     config: "no feature bfd\ndefault interface {{ test_int1 }}\ndefault interface\
```

Comment:
In some of the changes, including this, the `config` key is not replaced with `lines`. This doesn't throw an error because in nxos_config module, the `config` key is an alias for `running_config`. As a result, these tasks actually don't push changes to the device.

Reply:
Ah, good catch. I'll recheck and see what all I missed.